src/main/assembly/dist/cfg/ABR-reports.csv

```diff
@@ -0,0 +1,135 @@
+cd0aec40-e199-43d7-927b-ee12f1fe7a31,"ADC-rapport","adc[^a-z]+rapport[a-z]*"
+dfd7f0b8-3423-410e-ad10-5f740906c1da,"ADT-rapport","adt[^a-z]+rapport[a-z]*"
+351707da-ab1d-438e-9d42-915d785594b3,"Aeres-Milieu Rapport","aeres[^a-z]+milieu[^a-z]+rapport[a-z]*"
+8e84ebc8-f379-4056-b171-8fbf54a76f15,"Aestimatica-rapport","aestimatica[^a-z]+rapport[a-z]*"
+6826d9aa-2b50-4f90-bc8c-902c56d75d22,"Alkmaar, Monumentenzorg en Archeologie Gemeente","alkmaar[^a-z]+monumentenzorg[^a-z]+en[^a-z]+ar(ch|g)a?eologi[a-z][^a-z]+gemeente"
+05c754af-7944-4971-8280-9e1b4e474a8d,"Rapporten over Alkmaarse Monumenten en Archeologie","(rapporten[^a-z]+over[^a-z]+alkmaar[a-z][^a-z]+monumenten[^a-z]+en[^a-z]+ar(ch|g)a?eologi[a-z]([^a-z]*rama)?|rama)?"
```

Comment:
regular expression should but does not find matches as shown by `matches-per-rce.txt` generated with [TitlesSpec](https://github.com/DANS-KNAW/easy-convert-bag-to-deposit/blob/d934f9e2a05b4bcfcb9343400a9e5280f042fda9/src/test/scala/nl.knaw.dans.easy.bag2deposit/TitlesSpec.scala#L64)